Engine/Setup/BaseSetupHandler.cs

```diff
@@ -92,6 +93,9 @@ public static class BaseSetupHandler
                     cash.Update(data);
                 }
             });
+
+            Log.Trace("BaseSetupHandler.SetupCurrencyConversions(): " +
+                $"{string.Join(" | ", algorithm.Portfolio.CashBook.Values.Select(x => x.ToString()))}");
```

Comment:
Aww, you didn't like the nifty `CashBook.ToString()` overload, prints it all out as a table :)

Reply:
Whoops! Your right, the table looks much better, I'll replace it 👍